src/SentryLink.ts

```diff
@@ -94,7 +94,7 @@ export class SentryLink extends ApolloLink {
         error: (error) => {
           if (attachBreadcrumbs) {
             // We must have a breadcrumb if attachBreadcrumbs was set
-            (breadcrumb as GraphQLBreadcrumb).level = Severity.Error;
+            (breadcrumb as GraphQLBreadcrumb).level = 'error' as SeverityLevel;
```

Comment:
This type assertion should not be necessary - actually it covers up potential errors. Can you remove it?